packages/documentation/src/pages/architecture/front-end-standards/documented-decisions/summary.mdx

```diff
@@ -0,0 +1,9 @@
+# Summary of documented decisions
+
+- May 2016 - [Why we build with Ruby, rails, React, and Metalsmith](/architecture/front-end-standards/documented-decisions/language-and-tech-stack)
```

Comment:
I left the decisions out of the sidebar and tried to summarize them with their titles.